### ISSUE TITLE
Which parts of SophiaVL-R1-130k are used to train the reasoning model?

### ISSUE THREAD

**Issue by hxgr4ce** [kxfan2002/SophiaVL-R1#10]
I noticed that in `scripts/training_scripts/fullsets.yaml`, `data.train_files` is set to `Chart,OCR`, which means that only the data under the entries for `Chart` and `OCR` in `verl/data/dataset_info.json` are used to create the train dataloader. I also noticed that `General` is missing from `verl/data/dataset_info.json`. However, the paper says that _all_ of SophiaVL-R1-130k is used during RL.

Perhaps the training config files are outdated; do you mind clarifying the following?

1. To replicate your training, should we update `verl/data/dataset_info.json` to include the `General` category of SophiaVL-R1-130k, and `scripts/training_scripts/fullsets.yaml`'s `data.train_files` to include Chart, OCR, General, Knowledge, and Math?
2. Should we also update the learning rate listed in `scripts/training_scripts/fullsets.yaml`?
3. Did you use the samples found in `verl/data/demo.json` as the validation split?
4. Since you have a batch size of 16 and report training for 1500 steps, did the model really only see 1500*16=24000 out of 130000 samples?
